documentation/modules/rn-2.6.adoc

```diff
@@ -170,18 +166,28 @@ In some rare cases, the VM may be configured with a disk that is placed on a dat
 .Failure to migrate an image-based VM from {osp} to the `default` project
 
 The migration process fails when migrating an image-based VM from {osp} to the `default` project. link:https://issues.redhat.com/browse/MTV-964[(MTV-964)]
-For a complete list of all known issues in this release, see the list of link:https://issues.redhat.com/issues/?filter=12430275[Known Issues] in Jira.
+
+For a complete list of all known issues in this release, see the list of link:https://issues.redhat.com/issues/?filter=12435571[Known Issues] in Jira.
 
 
 [id="resolved-issues-26_{context}"]
 == Resolved issues
 
 This release has the following resolved issues:
 
+.VMs migrated from vSphere and OVA files are not being fully copied
+
+In {project-short} 2.6.0, there was a problem copying VMs from VMware vSphere and from OVA files, resulting in the failure of migrations using these source providers. The problem was resolved in {project-short} 2.6.1. link:https://issues.redhat.com/browse/MTV-1067[(MTV-1067)]
```

Comment:
this can be made more specific - the problem was specific to VMs with multiple disks. the migration appeared to succeed but all the disks were transferred to the same PV in the target environment - other disks were empty. therefore in some cases the VM couldn't boot (when the bootable disks was overridden) and in other cases, data from the other disks was missing